### DISCUSSION TITLE
Discussions not coming up on recent activity

### DISCUSSION THREAD

**Discussion by jackHedaya** [community/community#3162] [Discussions] [answered]
On the home page of the GitHub website, there's the recent activity tab. For some reason, discussions I'm involved in don't pop up there.

**Comment by PranamBhat** [comment, marked as answer]
Hi @jackHedaya 

Seems like your session has been timed out. Just refresh (CTRL + F5) the page.

Also, make sure you are following this guideline:

![image](https://user-images.githubusercontent.com/32120033/108161274-b2e99500-7110-11eb-8dda-4b757b52fe13.png)

**Reply by jackHedaya** [reply]
Ahh I thought it would also show discussions I was involved with. Guess it's a bug, not a feature.

**Reply by PranamBhat** [reply]
**True 😄  😄**
